packages/backend-modules/documents/lib/process.js

```diff
@@ -167,11 +167,43 @@ const processNodeModifiersInContent = (mdast, user) => {
   })
 }
 
+const processIfHasAccess = (mdast, user) => {
```

Comment:
can we add unit tests for this function? From reading the code I feel like if block without else and or else blocks at index 0 (falsy) might not be handled correctly

Reply:
Yeah, yeah, it's a bug!

Reply:
Yeah, yeah, we can tescht that.

Reply:
Fixed bug. Added rudimentary test.